spec/chirrup_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'chirrup'
```

Comment:
Might be worth having a test for the time that the chirrup was created? Look into Timecop and time.freeze

Reply:
This would be a good way to make my tests my comprehensive, will do!